operator/templates/pv-pvc.yaml

```
@@ -0,0 +1,30 @@
+metadata:
+  name: operator-controller-pvc
+  namespace: {{.Values.namespace}}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: hostpath
+  volumeMode: Filesystem
+  volumeName: operator-controller-pv
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)